### PR TITLE
Strip `v` from `kubelet` and `controlplane` version

### PR DESCRIPTION
**Pull request by masudur-rahman**
In some cases `kubelet` and `controlplane` versions are copied from `Kubernetes` version. `Kubernetes` version contains an extra `v` from `kubelet & controlplane` version. That's why I striped `v` from `kubelet` and `controlplane` version.

### CODE REVIEW

**Comment by detiber** [comment]
I don't know enough about this workflow to comment on this change.
/assign @justinsb

**Comment by justinsb** [comment]
This seems sensible to me, but I propose two things:

1) We avoid writing it back into the API object, but rather call stripVersion when we want to read the field
2) We discuss this in the cluster-api meeting on Wednesday.  We want this to be consistent across providers.  We could make this one of the first pieces of shared code in this area.  Alternatively we could validate a strict format e.g. "no v"; a reason for that would be to make it easier for other code to read the version from the Cluster/Machine object (without having to maintain the normalization logic)

File: pkg/cloud/google/machineactuator.go [hunk 1]
```diff
@@ -689,7 +693,14 @@ func (gce *GCEClient) updateMasterInplace(cluster *clusterv1.Cluster, oldMachine
 	return nil
 }
 
+func stripVersion(version string) (newVersion string) {
+	cutSet := "vV"
+	newVersion = strings.Trim(version, cutSet)
+	return
```
Comment (vincepri) [inline pkg/cloud/google/machineactuator.go]: This could be a one liner

```suggestion
	return strings.Trim(version, "vV")
```

**Comment by masudur-rahman** [comment]
Hi @justinsb, can you please recheck if the changes are okay ?

File: pkg/cloud/google/machineactuator.go [hunk 2]
```diff
@@ -689,6 +692,10 @@ func (gce *GCEClient) updateMasterInplace(cluster *clusterv1.Cluster, oldMachine
 	return nil
 }
 
+func stripVersion(version string) string {
+	return strings.Trim(version, "vV")
```
Comment (justinsb) [inline pkg/cloud/google/machineactuator.go]: Probably should be `TrimLeft`, but in practice it's not going to end with v anyway

**Comment by justinsb** [comment]
Thanks for the changes @masudur-rahman .  As I see it now, the primary case is to make sure that we don't specify a `v` in front of the version we pass to kubeadm or apt-get, and it looks narrowly scoped to that.

/approve
/lgtm

**Pull request merged**